utils/rollouts.py

```diff
@@ -5,18 +5,3 @@
 
 def owner_slug(owner: Owner) -> str:
```

Comment:
should this get deleted too?

Reply:
I think it's fine to leave this here, it might end up being useful next time someone wants to use a rollout

Reply:
I don't think so... that is a helper function, not directly related to this rollout. We can reuse that for future rollouts.
(even if there aren't none right now 😅)